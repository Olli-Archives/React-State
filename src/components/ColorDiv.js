import React from 'react';
import styles from './ColorDiv.css';
import PropTypes from 'prop-types';

export default function ColorDiv({ color }){
 

  console.log('props in color div', color);
  return (
    <div className={styles[color]}> 

    </div>
  );
}

ColorDiv.propTypes = {
  color: PropTypes.string.isRequired
}
;
