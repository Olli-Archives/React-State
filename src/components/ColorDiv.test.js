import React from 'react';
import { shallow } from 'enzyme';
import ColorDiv from './ColorDiv';

describe('colordiv component', ()=>{
  it('renders Random', ()=>{
    const wrapper = shallow(<ColorDiv/>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
