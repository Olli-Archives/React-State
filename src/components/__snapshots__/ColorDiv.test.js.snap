// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colordiv component renders Random 1`] = `
<div
  style={
    Object {
      "backgroundColor": undefined,
      "height": "100px",
      "width": "200px",
    }
  }
>
  im a div
</div>
`;
